great_expectations/datasource/fluent/schemas/SqliteDatasource.json

```diff
@@ -1,6 +1,6 @@
 {
     "title": "SqliteDatasource",
-    "description": "Adds a sqlite datasource to the data context.\n\nArgs:\n    name: The name of this sqlite datasource.\n    connection_string: The SQLAlchemy connection string used to connect to the sqlite database.\n        For example: \"sqlite:///path/to/file.db\"\n    assets: An optional dictionary whose keys are TableAsset names and whose values\n        are TableAsset objects.",
+    "description": "--Public API--Adds a sqlite datasource to the data context.\n\nArgs:\n    name: The name of this sqlite datasource.\n    connection_string: The SQLAlchemy connection string used to connect to the sqlite database.\n        For example: \"sqlite:///path/to/file.db\"\n    assets: An optional dictionary whose keys are TableAsset names and whose values\n        are TableAsset objects.",
```

Comment:
Why do we need to add this?

Reply:
[The `@public_api` decorator adds this prefix (and that's all it does)](https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/core/_docs_decorators.py#L16). I believe another step of the doc building process looks for this prefix when generating the docs? I haven't read that code.